core/src/torii/mod.rs

```diff
@@ -17,31 +17,41 @@ use warp::{
     Filter, Reply,
 };
 
-#[macro_use]
-mod utils;
-
 use crate::{
+    block::stream::{
+        BlockPublisherMessage, BlockSubscriberMessage, VersionedBlockPublisherMessage,
+        VersionedBlockSubscriberMessage,
+    },
     event::{Consumer, EventsSender},
     prelude::*,
     queue::{self, Queue},
     smartcontracts::{
         isi::query::{self, VerifiedQueryRequest},
         permissions::IsQueryAllowedBoxed,
     },
+    stream::{Sink, Stream},
     wsv::WorldTrait,
     Addr, Configuration, IrohaNetwork,
 };
 
-/// Main network handler and the only entrypoint of the Iroha.
-pub struct Torii<W: WorldTrait> {
+#[macro_use]
```

Comment:
```suggestion

```
AFAIK this is obsolete, see: https://stackoverflow.com/questions/26731243/how-do-i-use-a-macro-across-module-files

Reply:
is it? I think I saw you try to remove it in metrics PR?

Reply:
You know what? Backwards compatibility doesn't hurt.